Readme updates for 16.0 (Loris-MRI Ubuntu Readme)

christinerogers opened the pull request:
Updating the Loris-MRI Readme for 16.0 : 

* Now recommending manual install of minc toolkit via Vlad's webpage on GitHub
* On certain /data/ mounts (especially Ubuntu) the mkdir commands fail when running the imaging_install.sh script.  User needs to manually run these commands afterwards.  This typically affects all the subdirectories under /data/$PROJ/data/ 
* Release version stated in the Readme
* Adding instruction to enable apache writability on data directories for Imaging Uploader

MounaSafiHarab commented on README.md:
````diff
@@ -47,13 +51,18 @@ See aces/Loris README.md for further information and Loris installation informat
+   ```source /data/$projectname/bin/mri/environment```
+
+   Ensure that the apache envvars file includes all the EXPORT statements from minc-toolkit-config.sh (this file located in the path where the MINC tools are installed). This is necessary for BrainBrowser to successfully load MINC images.
 
````
@christinerogers,

this is no longer necessary due to the latest changes in this pull request:
https://github.com/aces/Loris/pull/1843
But instead, we can specify that the config.xml file (on the Loris side) should include, as documented in the above pull request:
  <!-- MINC TOOLS PATH -->
    <MINCToolsPath>/opt/minc/</MINCToolsPath>
(where /opt/minc/ is the MINC tools path in this example).

christinerogers commented:
@MounaSafiHarab  I've updated this pull request -- it should cover what #125 intended, and also the www-data writability solution.  

Please review - and note I've removed the instruction to include source /data/$projectname/bin/mri/environment in the bashrc -- is that correct per our discussion?   thanks!

MounaSafiHarab commented on README.md:
```diff
@@ -39,21 +50,40 @@ See aces/Loris README.md for further information and Loris installation informat
+  If the imaging install script reports errors in creating directories (due to /data/ mount permissions), manually execute mkdir and chmod commands starting at [imaging_install.sh:L90](https://github.com/aces/Loris-MRI/blob/16.0-dev/imaging_install.sh#L90)
+
+  The installer will make apache (www-data) part of the lorisadmin linux group.  
+
```
@christinerogers,

Hi Christine,
so this now needs to be updated, as we are now changing the group to be apache (www-data) and then using the chmod g+s;
https://github.com/aces/Loris-MRI/pull/129

christinerogers commented:
@MounaSafiHarab any further comments? thanks!
